src/tasks/init.js

```diff
@@ -22,18 +22,22 @@ const path = require('path');
 const files = require('./constants/files');
 
 module.exports = () => {
-  [
-    files.CONTAINER_FILENAME,
-    files.LIB_SANDBOX_HTML_FILENAME
-  ].forEach((filename) => {
-    fs.copy(
-      path.resolve(files.CLIENT_SRC_PATH, filename),
-      path.resolve(files.CONSUMER_CLIENT_SRC_PATH, filename),
-      {
-        clobber: false
-      }
-    );
-  });
+  const descriptor = require(path.resolve(files.EXTENSION_DESCRIPTOR_FILENAME));
```

Comment:
The `init` command will create the `.sandbox` folder only for web extensions.

Reply:
We need to update the readme to reflect this: https://github.com/Adobe-Marketing-Cloud/reactor-sandbox/tree/f18c51a19606ba03c4c4be82e779830330fa9276#configuring-the-sandbox